mobile/examples/basic_usage/model/gen_model.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+OUTPUT_DIR=${1:?"Please specify an output directory."}
+
+# Get directory this script is in
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+mkdir -p ${OUTPUT_DIR}
+cd ${OUTPUT_DIR}
+
+python3 ${DIR}/single_add_gen.py
+python3 -m onnxruntime.tools.convert_onnx_models_to_ort --optimization_level basic .
```

Comment:
Remove the `--optimization_level basic` or add some comments?
See https://github.com/microsoft/onnxruntime/pull/7941#discussion_r645328442